partner_label/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Therp BV <http://therp.nl>
```

Comment:
Please change http: to https: in copyright and license lines troughout the module.